scenes/game/card_base.gd

```diff
@@ -439,3 +446,104 @@ func _on_button_mouse_exited() -> void:
 
 func _on_attachment_button_pressed() -> void:
 	view_attachments.emit(_attached_cards)
+
+func _update_no_proxy_panel(card: CardBase) -> void:
```

Comment:
What are your thoughts on this functionality being contained in its own scene and CardBase just having one of these?
Especially if we think we're going to add significantly more code.
If not, maybe it's fine.

Reply:
I agree with separating it to its own scene since ``CardBase`` responsibility here is just to toggle its visibility and supplying the data.